tsconfig.json

```
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```

We can target `esnext` since we are going to run this in a modern Node/Bun environment. 

```suggestion
    "target": "esnext",
```